task22/js/main.js

```diff
@@ -68,13 +68,17 @@ const getCreatedTableHeader = (userTableColumn) => {
     const theadElement = document.createElement("thead");
     const tableRowElement = document.createElement("tr");
     const tableHeaderFragment = document.createDocumentFragment();
-    for(const column of Object.values(userTableColumn)){
+    const sortTarget = ["id", "age"];
```

Comment:
nits
Since the column name information is objected to, I thought it would be more maintainable to bring that information.

```suggestion
    const sortTarget = [userTableColumn.id, userTableColumn.age];
```

However, we did not have to use the
```
if (sortTarget.includes(columnKey)){
    tableHeaderElement.appendChild(renderSortButtons);
}
```
I may need to change the implementation here a bit (unconfirmed), so I'll leave the fix to you.

Reply:
I didn't have that perception, so I learned a lot. However, I will make use of it in my next implementation. Thank you for your suggestion.